### ISSUE TITLE
Timeout reached on SELECT * FROM Win32_UserAccount

### ISSUE THREAD

**Issue by MarcSamD** [glpi-project/glpi-agent#379]
### Bug reporting acknowledgment

Yes, I read it

### Professional support

Yes, I know

### Describe the bug

The inventory takes a very long time because WMI queries timeout on Win32_UserAccount for different accounts:

```
[Wed Apr 19 14:04:54 2023][info] running task Inventory
[Wed Apr 19 14:04:54 2023][info] New inventory from computername.domainname.org-2023-04-18-17-54-58 for server0
[Wed Apr 19 14:17:37 2023][info] Timeout reached on SELECT * FROM Win32_UserAccount WHERE Name='Administrator' AND Disabled='False' and Lockout='False' WMI query
[Wed Apr 19 15:07:25 2023][info] Timeout reached on SELECT * FROM Win32_UserAccount WHERE Name='john.snow' AND Disabled='False' and Lockout='False' WMI query
```
Note that the computer is member of an AD domain and "john.snow" is a domain user.

Seems to be the same issue as https://github.com/glpi-project/glpi-agent/issues/232 which is closed.

### To reproduce

In my case, any inventory will reproduce this error, which is new and following the migration from FusionInventory agent to GLPI-Agent 1.4 then 1.5-git3c8816ae

### Expected behavior

Inventory taking only a few minutes

### Additional context

GLPI 10.0.7
GLPI Inventory 1.2.1
GLPI-Agent 1.5-git3c8816ae on Windows 10

**Comment by g-bougard** [comment]
Hi @MarcSamD 

the fix for #232 has been extended in current nightly after #360 & #374. Have checked that issues too ?

Can you confirm you're also using `scan-profiles` option ? And what's the value of `backend-collect-timeout` you configured for that agent ?
Can you enabled `debug=2`, restart the agent and share the log of the inventory in that debug level (not including the generated inventory) ?

Is the problem only on one or few specific computers or on all computers ?

**Comment by g-bougard** [comment]
Another question, do you still have the error if you set `no-category` to `user` in registry and restart the service ?

**Comment by MarcSamD** [comment]
Ok silly me, I was still in 1.4... I used glpi-agent.vbs to try to upgrade to 1.5 nightly and I thought it was successful but this is another issue...

So I confirm that by really using the latest GLPI-Agent 1.5-git3c8816ae (upgrading via .zip this time), I don't have any issue and the inventory took less than 1 minute.

Thank you @g-bougard for the quick reply and sorry for the trouble.
